src/fragments/lib/auth/js/manageusers.mdx

```diff
@@ -4,45 +4,150 @@ This section covers the client-side development experience for password and user
 
 ### Change password
 
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { Auth } from 'aws-amplify';
+import type { CognitoUser } from "@aws-amplify/auth";
```

Comment:
@indieisaconcept, we are considering importing CognitoUser directly from the library. Can you remove this line and any references to this type as well?